### DISCUSSION TITLE
P1001: Can't reach database server at `ec2-3-229-127-203.compute-1.amazonaws.com`:`5432`

### DISCUSSION THREAD

**Discussion by iradukundairenee** [prisma/prisma#10608] [Q&A]
guyz i'm trying to create table in my data base  npx prisma db push command but there are giving me the error  any help please?
![12](https://user-images.githubusercontent.com/62067881/145037219-e1d56f8b-7f9b-4149-8bcd-db7e0e57edca.png)

**Comment by janpio** [comment]
Is that database server reachable from your computer? Can you connect using a database UI?
